tools/breaking-changes/data/breaking-changes.json

```diff
@@ -22415,7 +22415,7 @@
     "type": "MemoizedSelector<StateWithCheckout, ORDER_TYPE>",
     "isDeleted": true,
     "deletedComment": "Variable CheckoutSelectors.getSelectedOrderType has been removed and is no longer part of the public API.",
-    "migrationComment": "",
+    "migrationComment": "If you are not using scheduled-replenishment feature, you don't need this variable any longer. Otherwise, it has been moved to 'currentOrderType' in 'CheckoutScheduledReplenishmentPlaceOrderComponent' (imported from '@spartacus/checkout/scheduled-replenishment/components')",
```

Comment:
I believe you should mention to get it from CheckoutReplenishmentFormService in @spartacus/checkout/scheduled-replenishment/components using getOrderType